packages/perseus/src/components/math-input.jsx

```diff
@@ -1,174 +1,50 @@
-/* eslint-disable react/sort-comp */
 // @flow
 import classNames from "classnames";
 import $ from "jquery";
 import MathQuill from "mathquill";
-import PropTypes from "prop-types";
 import * as React from "react";
 import ReactDOM from "react-dom";
 import _ from "underscore";
 
 import TexButtons from "./tex-buttons.jsx";
 
-const PT = PropTypes;
+import type {ButtonSetsType} from "./tex-buttons.jsx";
+
+type ButtonsVisibleType = "always" | "never" | "focused";
+
+type Props = {|
+    className?: string,
+    value: string,
+    onChange: $FlowFixMe,
```

Comment:
Is the comment below about this `onChange` prop?  If so, we typically spread in the `ChangeableProps` type from `changeable.jsx` for this. 
```suggestion
    ...ChangeableProps,
```